src/en/web/design/forms.md

```diff
@@ -39,3 +40,47 @@ Lastly, the wording of the error messages should be explicit.
 
 **Don’t:**  
 ![screenshot of a form displaying irrelevant error messages](../../images/formulaire-ko.png)
+
+## Limit redundant entries
+
+**Target:**  People with cognitive disabilities (especially with memory difficulties).
+
+**When:** During design and development.
+
+**Description:**  
+The intent of this criteria is to reduce the cognitive effort required by filling out forms with redundant entries.
+
+People with memory problems may have difficulty if a form asks to enter information that has already been entered before. For example: having to enter your email address a second time.
+ 
+Forcing the user to enter the same information several times can cause stress or cognitive fatigue with the risk of generating incorrect entries.
+The user may find it impossible to continue the customer journey, or even abandon the task.
+
+Information previously entered by or provided to the user that is required to be entered again in the same process is either auto-populated, or available for the user to select.
+
+**Checklist:**
+In a process that requires entering information several times (for example multi-step form), check that at least one of following conditions is met:
+
+* Fields whose data has already been entered are filled automatically.
+* User can select data already entered from a list.
+* User can check a box to automatically fill a form with data already entered.
+
+Using an auto-completion system is not considered sufficient.
```

Comment:
WCAG: 'The autocomplete feature of browsers is not considered sufficient' (https://www.w3.org/WAI/WCAG22/Understanding/redundant-entry.html)

autocomplete from the website = autocomplete from the browser?